creation/web_base/create_temp_mapfile.sh

```diff
@@ -15,10 +15,10 @@
 glidein_config="$1"
 
 # import add_config_line function
-add_config_line_source="`grep '^ADD_CONFIG_LINE_SOURCE ' "$glidein_config" | cut -d ' ' -f 2-`"
+add_config_line_source=$(grep -m1 '^ADD_CONFIG_LINE_SOURCE ' "$glidein_config" | cut -d ' ' -f 2-)
 source "$add_config_line_source"
```

Comment:
Add the shellcheck line:
```
# shellcheck source=./add_config_line.source
```
prefer `.` to `source`

Reply:
updated